content/en/continuous_integration/pipelines/teamcity.md

```diff
@@ -0,0 +1,95 @@
+1. Download the [Datadog CI Integration plugin][5] on the TeamCity server by going to
+Administration -> Plugins -> Browse Plugin Repository.
+2. For the integration to work properly, a [TeamCity composite build][6] needs be added as the last build
+of the build chain (this step can be skipped if there is already one). Namely, this build must have
+a dependency on the current last build of the chain and no other builds depending on it. Build chains not having a
+composite build as their last build are ignored by the plugin.
+3. The following configuration parameters need to be present for the TeamCity projects:
+   * `datadog.ci.api.key`: having as value your [Datadog API Key][2].
+   * `datadog.ci.site`: having as value {{< region-param key="dd_site" code="true" >}}.
+   * `datadog.ci.enabled`: having as value `true` (`false`
+   can be used to disable the plugin for a specific project).
+
+   These configuration parameters can be added either at TeamCity subprojects level
+   or at the [TeamCity Root Project][10] level. When added at the Root project level, they are propagated
+   to all the subprojects. For example, to enable the plugin for all projects, `datadog.ci.enabled` can be added with
+   value `true` to the Root Project. More information on defining configuration parameters
+   are available in the [TeamCity Project Hierarchy][9] docs.
+4. Enable the plugin by restarting the TeamCity server.
+
+## Visualize pipeline data in Datadog
+
+The [Pipelines][3] and [Pipeline Executions][4] pages populate with data after the pipelines finish.
+
+**Note**: The Pipelines page shows data for only the default branch of each repository.
+
+## Troubleshooting
+
+All the logs generated by the Datadog CI Integration plugin are stored inside the `teamcity-server.log` file and can be
+accessed from the TeamCity Server by going to "Administration -> Diagnostic -> Server Logs". If the plugin is not behaving
+correctly, these logs can be checked to get additional context on what might be wrong.
+
+## Git user information
+
+The plugin retrieves the Git author and committer information based on the [TeamCity username style][7].
+TeamCity has two username styles that do not contain the user email in them: "UserId" or "Author Name". When
+one of these styles is used, the plugin automatically generates an email for the user by adding `@Teamcity`
+at the end of the username.
+For example, the plugin generates `john.doe@Teamcity` as the Git author email when the "UserId" username
+style is used and the git author username is `john.doe`.
+
+<div class="alert alert-info"><strong>Note:</strong> The git author email is used for
+<a href="https://www.datadoghq.com/pricing/?product=ci-visibility#ci-visibility" target="_blank">billing</a> purposes,
+therefore there might be cost implications when either "UserId" or "Author Name" are used as TeamCity username styles.
+Reach out to us if you have any questions about your use case.</div>
```

Comment:
How can I say "Reach out to us" without incurring in these warnings? Contact Datadog support?